Documentation/Development.md

```diff
@@ -63,6 +63,8 @@ using a [snapshot](https://swift.org/download/#releases) from swift.org.
 	$ cd swiftpm
 	$ Utilities/bootstrap
 	```
+
+    Note: The bootstrap script requires having [Ninja](https://ninja-build.org/) installed.
```

Comment:
@noahsark769, you might want to refer to the relevant docs from the main Swift repo: https://github.com/apple/swift/blob/master/README.md#macos

Reply:
Sure, good idea!